plugins/gatsby-transformer-versions-yaml/gatsby-node.js

```diff
@@ -20,10 +20,12 @@ exports.onPostBuild = async ({store}) => {
 
   // versions.yml structure is [{path: string, url: string, ...}, ...]
   createRedirects(
-    versions.filter(version => version.path && version.url).map(version => ({
-      fromPath: version.path,
-      toPath: version.url,
-    })),
+    versions
+      .filter(version => version.path && version.url)
```

Comment:
Is this automatically done? Just change the code style, but why?

Reply:
This part changes for split function in chain to new line. I think the code looks better now.